advanced-integration/paypal-api.js

```diff
@@ -60,7 +69,10 @@ export async function generateAccessToken() {
   return jsonData.access_token;
 }
 
-// generate client token
+/**
+ * Generate a client token
+ * @see https://developer.paypal.com/docs/multiparty/checkout/advanced/integrate/#link-sampleclienttokenrequest
```

Comment:
I think we can use the regular docs page here (the multiparty links are technically for partners)
```suggestion
 * @see https://developer.paypal.com/docs/checkout/advanced/integrate/#link-sampleclienttokenrequest
```

Reply:
Hey @jshawl could you make this url change when you get a chance?